test/format/ops.c

```diff
@@ -230,12 +230,20 @@ tinfo_teardown(void)
 static void
 rollback_to_stable(WT_SESSION *session)
 {
+    u_int num_threads;
+    char cfg[512];
+
     /* Rollback-to-stable only makes sense for timestamps. */
     if (!g.transaction_timestamps_config)
         return;
 
-    /* Rollback the system. */
-    testutil_check(g.wts_conn->rollback_to_stable(g.wts_conn, NULL));
+    /*
+     * Rollback the system using up to 10 threads. Extend to 11 values to cover the NULL config
+     * case.
+     */
+    num_threads = mmrand(&g.extra_rnd, 0, 11);
+    testutil_snprintf(cfg, sizeof(cfg), "treads=%" PRIu32, num_threads);
```

Comment:
`treads` should be `threads`. Have you tested this? Because this typo should have caused a failure.

Is there something special about 10 threads? Could you explicitly set 0-9 and have 10 mean NULL? Most other mmrand calls use == for the special case. So `num_threads == 10` ? NULL : cfg`. That way even if someone changes it to 20 threads maximum, then only one case results in a NULL config, not many/most.

Reply:
> treads should be threads. Have you tested this? Because this typo should have caused a failure.

Good catch, thank you! I relied on the PR testing, which is currently all red because of this error.

There is nothing specific about the value 10, it is just another possible value RTS should be able to deal with. Instead of using `> 10`, I should have used `== 11`:
```
    testutil_check(g.wts_conn->rollback_to_stable(g.wts_conn, num_threads > 10 ? NULL : cfg)); # Should be replaced with == 11
```
If we ever change the number of threads that can be used for RTS, I expect this code to be updated accordingly anyway to cover the new set of values